src/coreclr/pal/src/libunwind/src/mi/mempool.c

```diff
@@ -127,7 +127,7 @@ mempool_init (struct mempool *pool, size_t obj_size, size_t reserve)
     }
 
   pool->obj_size = obj_size;
-  pool->reserve = reserve;
+  pool->reserve = (unsigned int)reserve;
```

Comment:
Should we instead change type of `reserve` and `num_free` to `size_t`?
https://github.com/dotnet/runtime/blob/eb57c1276add1ec7e35897bfdbbebb648839dee3/src/coreclr/pal/src/libunwind/include/mempool.h#L66-L67

Reply:
Not unreasonable but it does change the size of the allocation now and I'm not sure that is desirable. I made the narrowest impacting change. Looking for guidance. Perhaps submit PR to libunwind and look for suggestions there?

Reply:
> Perhaps submit PR to libunwind and look for suggestions there?

Sounds reasonable. This code is there for 20+ years (https://github.com/libunwind/libunwind/commit/dd3c681c6a60f6f26125d6a7ce42f53c54a41101), code analyzers are improved a lot since. :)

Reply:
Oh wow. Good call.